Week-1/README.m

```diff
@@ -0,0 +1,47 @@
+Web browser are most used software The main function of a browser is to present the web resource you choose, by requesting it from the server and displaying it in the browser window.The resource can be HTML document or PDF , location is URI (Uniform Resource Identifier)
```

Comment:
file name is incorrect. It should be `README.md`

Reply:
corrected the filename and added the description with PR